test/commands/build.test.ts

```diff
@@ -64,7 +64,7 @@ describe('build', function () {
     });
     const tmp_config_file = path.join(tmp_dir, ARCHITECTPATHS.CLI_CONFIG_FILENAME);
     fs.writeJSONSync(tmp_config_file, config);
-    const app_config_stub = sinon.stub().resolves(new AppService(tmp_dir));
+    const app_config_stub = sinon.stub().resolves(new AppService(tmp_dir, '0.0.1'));
```

Comment:
Will these need to be changed as the CLI gets updated or is `0.0.1` more of a "mock version" for testing purposes only?

My only comment would be that we consolidate these references if we're having to update them frequently, but I think this pattern is fine if they stay `0.0.1` forever.

Reply:
I don't really see why we would need to change them since these are just tests, and this isn't related to any of the tests